### PR TITLE
lint legacy ibc testing

### PR DESCRIPTION
**Pull request by faddat**
I apologize, because `gh` has no such preview tab. 

This PR lints the legacy_ibc_testing folder.

Please go the the `Preview` tab and select the appropriate sub-template:

* [Production code](?expand=1&template=production.md) - for types `fix`, `feat`, and `refactor`.
* [Docs](?expand=1&template=docs.md) - for documentation changes.
* [Others](?expand=1&template=others.md) - for changes that do not affect production code.

### CODE REVIEW

File: legacy_ibc_testing/core/events.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func ReconstructPacketFromEvent(event abci.Event) (packet types.Packet, err erro
 		return packet, err
 	}
 	return types.NewPacket(
-		attrMap[string(types.AttributeKeyData)], // data
+		attrMap[string(types.AttributeKeyDataHex)], // data
```
Comment (smarshall-spitzbart) [inline legacy_ibc_testing/core/events.go:39]: What does this change do? Doesn't seem like a lint

File: legacy_ibc_testing/testing/events.go [hunk 2]
```diff
@@ -69,7 +69,7 @@ func ParseAckFromEvents(events sdk.Events) ([]byte, error) {
 	for _, ev := range events {
 		if ev.Type == channeltypes.EventTypeWriteAck {
 			for _, attr := range ev.Attributes {
-				if string(attr.Key) == channeltypes.AttributeKeyAck {
+				if attr.Key == channeltypes.AttributeKeyAckHex {
```
Comment (smarshall-spitzbart) [inline legacy_ibc_testing/testing/events.go:72]: Same question as above. Maybe the two changes are tied together but I'm evaluating this as a linting / fully refactor PR

**Comment by smarshall-spitzbart** [comment]
These changes seem to make tests fail

File: legacy_ibc_testing/core/events.go (hunk 1 above)
Comment (faddat) [reply legacy_ibc_testing/core/events.go:39]: old call is deprecated and I think that these both return the same thing

**Comment by faddat** [comment]
ah, you know, that explains some stuff.  Also, it is in the exact location you flagged -- I thought the two calls did the same thing.  

Thanks!

**Comment by faddat** [comment]
PS: do you know where to find the preview tab?

**Comment by mpoke** [comment]
> PS: do you know where to find the preview tab?

When you open a PR using the github UI (https://github.com/cosmos/interchain-security/compare), you have a `Write` tab, which is selected by default. Next to it, there is the `Preview` tab.

**Comment by faddat** [comment]
Thanks @mpoke -- is it posssible to settle on one template?

I'll fix this one up manually, but basically here's my usual workflow, which is why you're still sometimes seeing this from me:


```bash
git checkout origin/main
git switch -c faddat/branchname
# do stuff
git commit -a
gh pr create --draft
# do more stuff
git commit -a
gh pr ready 1078
```

**Review by smarshall-spitzbart — APPROVED** [review]

**Review by jtremback — APPROVED** [review]
LGTM

**Comment by mpoke** [comment]
> Thanks @mpoke -- is it posssible to settle on one template?

Having different templates for different contributions types helps with the workflow. For example, a bug fix has different requirements than an update of the docs. 
 
> 
> I'll fix this one up manually, but basically here's my usual workflow, which is why you're still sometimes seeing this from me:
> 
> ```shell
> git checkout origin/main
> git switch -c faddat/branchname
> # do stuff
> git commit -a
> gh pr create --draft
> # do more stuff
> git commit -a
> gh pr ready 1078
> ```

I'm not that familiar with the `gh` CLI command, but here is a related issue https://github.com/cli/cli/issues/2567.

**Comment by faddat** [comment]
awesome, just a moment sir.

**Pull request merged**